record-and-playback/presentation/scripts/process/presentation.rb

```diff
@@ -257,19 +244,18 @@
     metadata = Nokogiri::XML(File.read("#{target_dir}/metadata.xml"))
     ## Update status
     recording = metadata.root
-    state = recording.at_xpath("state")
-    state.content = "processed"
+    state = recording.at_xpath('state')
+    state.content = 'processed'
     ## Write the new metadata.xml
-    metadata_file = File.new("#{target_dir}/metadata.xml","w")
+    metadata_file = File.new("#{target_dir}/metadata.xml", 'w')
     metadata_file.write(metadata.root)
     metadata_file.close
-    BigBlueButton.logger.info("Created an updated metadata.xml with state=processed")
-
-  rescue Exception => e
+    BigBlueButton.logger.info('Created an updated metadata.xml with state=processed')
+  rescue StandardError => e
     BigBlueButton.logger.error(e.message)
     e.backtrace.each do |traceline|
       BigBlueButton.logger.error(traceline)
     end
     exit 1
   end
-end
+end
```

Comment:
You editor appears to be misconfigured to remove the trailing newline on text files, please fix this.

Reply:
You're right, it's enabled now.